releng/releng-target/xtext-core.target.target

```diff
@@ -9,5 +9,9 @@
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="false" type="InstallableUnit">
 			<repository location="http://download.eclipse.org/releases/luna/201502271000/"/>
 		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="false" type="InstallableUnit">
+			<repository location="http://download.eclipse.org/modeling/tmf/xtext/updates/orbit/"/>
+			<unit id="org.junit" version="0.0.0"/>
+		</location>
```

Comment:
If you edit this file directly, the change will be overwritten the next time the p2 build is generated. You should add the dependency in [p2-deployment.gradle](https://github.com/eclipse/xtext-core/blob/master/gradle/p2-deployment.gradle) instead.